.github/workflows/gradle.yaml

```diff
@@ -179,3 +209,75 @@ jobs:
         with:
           name: SlimeVR-GUI-MacDmg
           path: target/release/bundle/dmg/slimevr.dmg
+
+      - name: Prepare for release
+        if: startsWith(github.ref, 'refs/tags/')
+        run: |
+          cp target/release/bundle/dmg/slimevr.dmg ./SlimeVR-mac.dmg
+
+      - name: Upload to draft release
+        uses: softprops/action-gh-release@v1
+        if: startsWith(github.ref, 'refs/tags/')
+        with:
+          draft: true
+          generate_release_notes: true
+          files: |
+            ./SlimeVR-mac.dmg
+
+
+  bundle-windows:
+    runs-on: windows-latest
+    needs: build
+    if: contains(fromJSON('["workflow_dispatch", "create"]'), github.event_name)
+    steps:
+      - uses: actions/checkout@v3
+        with:
+          submodules: recursive
+
+      - uses: actions/download-artifact@v3
+        with:
+          name: "SlimeVR-Server"
+          path: server/build/libs/
+
+      - name: Cache cargo dependencies
+        uses: Swatinem/rust-cache@v2
+        with:
+          shared-key: "windows-latest"
+
+      - name: Use Node.js
+        uses: actions/setup-node@v3
+        with:
+          node-version-file: '.node-version'
+          cache: 'npm'
+
+      - name: Build
+        run: |
+          npm ci
+          npm run skipbundler
+      
+      - name: Bundle to zips
+        shell: bash
+        run: |
+          mkdir SlimeVR
+          cp gui/src-tauri/icons/icon.ico ./SlimeVR/run.ico
+          cp server/build/libs/slimevr.jar ./SlimeVR/slimevr.jar
+          cp server/resources/* ./SlimeVR/
+          cp target/release/slimevr.exe ./SlimeVR/
+          7z a -tzip SlimeVR-win64.zip ./SlimeVR/
+
+          mv ./SlimeVR/slimevr.exe ./SlimeVR/slimevr-ui.exe
+          7z a -tzip SlimeVR.zip ./SlimeVR/
```

Comment:
What is this zip for if it's not used anywhere?

Reply:
sorry, i still havent finished this. Thanks for the review tho <3